.github/docker/Dockerfile-musllinux.template

```diff
@@ -23,7 +23,9 @@
 #
 FROM quay.io/pypa/musllinux{{ TYPE }}_{{ ARCH }}:latest
 
-COPY requirements.txt /tmp/requirements.txt
+#COPY requirements.txt /tmp/requirements.txt
+COPY requirements_full.txt /tmp/requirements_full.txt
+COPY requirements_full.txt /tmp/requirements_dev.txt
```

Comment:
```suggestion
COPY requirements_full.txt requirements_dev.txt /tmp/
```

Reply:
The original lines here also (incorrectly) copies over `requirements_full.txt` twice.

Reply:
it is correct in the PR

Reply:
ok, combined them in one command